src/main/java/frc/robot/commands/autonomousCommands/TrajectoryTutCommand.java

```diff
@@ -0,0 +1,103 @@
+
+public class TrajectoryTutCommand extends SequentialCommandGroup {
+    
+    private Drive drive;
+    private DifferentialDriveVoltageConstraint autoVoltageConstraint; 
+    private TrajectoryConfig config;
+    private Trajectory exampleTrajectory;
+    private RamseteCommand ramseteCommand;
+    
+    private final double kMaxSpeedMetersPerSecond = 0.5;
+    private final double kMaxAccelerationMetersPerSecondSquared = 0.1;
+    private final double kRamseteB = 0.0;
+    private final double kRamseteZeta = 0.0;
+    
+    public TrajectoryTutCommand(Drive drive) {
+         // Create a voltage constraint to ensure we don't accelerate too fast
+        this.drive = drive;
+        this.autoVoltageConstraint = new DifferentialDriveVoltageConstraint(
+            new SimpleMotorFeedforward(
+                DriveConstants.ksVolts,
+                DriveConstants.kvVoltSecondsPerMeter,
+                DriveConstants.kaVoltSecondsSquaredPerMeter),
+            DriveConstants.kDriveKinematics,
+            10);
+
+        this.config = new TrajectoryConfig(
+            this.kMaxSpeedMetersPerSecond,
+            this.kMaxAccelerationMetersPerSecondSquared)
+            // Add kinematics to ensure max speed is actually obeyed
+            .setKinematics(DriveConstants.kDriveKinematics)
+            // Apply the voltage constraint
+            .addConstraint(autoVoltageConstraint); 
+
+        this.exampleTrajectory = TrajectoryGenerator.generateTrajectory(
+                // Start at the origin facing the +X direction
+                new Pose2d(0, 0, new Rotation2d(0)),
+                // Pass through these two interior waypoints, making an 's' curve path
+                List.of(new Translation2d(1, 1), new Translation2d(2, -1)),
+                // End 3 meters straight ahead of where we started, facing forward
+                new Pose2d(3, 0, new Rotation2d(0)),
+                // Pass config
+                config);
+        
+        this.ramseteCommand = new RamseteCommand(
+                    exampleTrajectory,
+                    this.drive::getPose,
+                    new RamseteController(),//(this.kRamseteB, this.kRamseteZeta),
+                    new SimpleMotorFeedforward(
+                        DriveConstants.ksVolts,
+                        DriveConstants.kvVoltSecondsPerMeter,
+                        DriveConstants.kaVoltSecondsSquaredPerMeter),
+                    DriveConstants.kDriveKinematics,
+                    this.drive::getWheelSpeeds,
+                    new PIDController(DriveConstants.kPDriveVel, 0, 0),
+                    new PIDController(DriveConstants.kPDriveVel, 0, 0),
+                    // RamseteCommand passes volts to the callback
+                    this.drive::tankDriveVolts,
+                    this.drive);
+                        this.drive.resetOdometry(exampleTrajectory.getInitialPose());
+                        addCommands(this.ramseteCommand);
```

Comment:
The formatting on 78 and 79 here make it look like they are part of the parameters to the RamseteCommand constructor. Adding a newline and changing the indentation can help. Again, it's just a readability issue, but it's good to get in the habit.